packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -459,6 +460,8 @@ export const prepareVault = (baggage, marshaller, zcf) => {
           seat.exit();
           helper.assignPhase(Phase.CLOSED);
           helper.updateDebtSnapshot(helper.emptyDebt());
+
+          // XXX causes error in 'price drop'. Is stream already closed?
```

Comment:
needs to be fixed before merge
```suggestion
          // FIXME DONOTMERGE causes error in 'price drop'. Is stream already closed?
```

Reply:
I looked into this and didn't figure anything out. Can you help diagnose?